wordpress-mobile/WordPress-iOS
# Update help shift to 6.2.0

Fixes #7694 

Update Helpshift SDK to 6.20

I continue to use the approach of copying the library to the vendors file.

To test:
 - Start the app
 - Go to Me -> Help & Support -> Contact Us
 - Check that chat is working ok.
 - Check that ticket deflection is working ok.


Needs review: @astralbodies and @elibud
